webnode/package.json

```diff
@@ -9,7 +9,7 @@
     "chalk": "1.1.3",
     "css-loader": "0.28.7",
     "curl.lib.js": "^1.0.22",
-    "datamap-generator": "git+https://github.com/oysterprotocol/datamap-generator.git",
```

Comment:
@tenisakb I believe we want to keep the reference the way it was. The last build that i saw that was failing was because of an outdated package. You should be able to set the package back the way it was and do a yarn install or yarn upgrade without hitting an error

Reply:
If it won't work the old way then leave it the way you have it.  But it's weird that it is causing errors, does anyone know if Travis was causing errors with this before (although I'm not sure if we had Travis enabled at a time when the repo had `"git+https://github.com/oysterprotocol/datamap-generator.git"`)?